var/spack/repos/builtin/packages/r/package.py

```diff
@@ -82,6 +82,8 @@ class R(Package):
     depends_on('pcre')
     depends_on('jdk')
 
+    patch('zlib.patch', when='@:3.3.1', level=0)
```

Comment:
My low-tech way to make the patch was to stage the package, cd into the stage dir/R-3.3.1, save a copy of the configure script, fix it, then diff the '.orig' to the new one.  That resulted in a patch that requires `level=0`.  I could edit the patch and monkey around with the file names so that it runs at `level=1`, but that seems disingenuous.  Is it worth it?